src/js/components/DataTable/__tests__/DataTable-test.tsx

```diff
@@ -2073,8 +2073,9 @@ describe('DataTable', () => {
 
     expect(container.firstChild).toMatchSnapshot();
 
-    fireEvent.mouseEnter(getByText('alpha'));
-    fireEvent.keyDown(getByText('alpha'), {
+    const row = screen.getByText('alpha').closest('tr');
+    fireEvent.focus(row!);
```

Comment:
What does the `!` mark do here?

Reply:
It's a shorthand to mean "not null" -- so it'll only try to focus if row exists.